src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -1686,12 +1686,151 @@ def floordiv():
     @classmethod
     def from_dict():
         """
-        Construct ``DataFrame`` from dict of array-like or dicts.
+        Construct DataFrame from dict of array-like or dicts.
+
+        Creates DataFrame object from dictionary by columns or by index allowing dtype specification.
+
+        Parameters
+        ----------
+        data : dict
+            Of the form {field : array-like} or {field : dict}.
```

Comment:
what is an example of `{field: dict}` ?  I do not find one example below to show this

Reply:
This is if you want to specify both the column labels and the row labels using a nested dict. Added a test for it.